Index: build/configure.ac
===================================================================
--- build.orig/configure.ac
+++ build/configure.ac
@@ -239,30 +239,6 @@ if test "x$enable_flac_plugin" = xyes;th
 fi
 
 #################################################################
-# Check for cutter
-#################################################################
-
-AM_CONDITIONAL(HAS_CUTTER, false)
-
-AC_ARG_ENABLE(cutter, [AC_HELP_STRING([--disable-cutter],[ Disable Cutter unit testing. ]) ],
-    [enable_cutter=$enableval],[enable_cutter="yes"])
-
-if test "x$enable_cutter" = xyes;then
-
-AC_CHECK_CUTTER
-
-cutter_command="no"
-AC_CHECK_FILE([$CUTTER], [cutter_command="yes"])
-
-if test "x$CUTTER" != x;then
-  if test "x$cutter_command" = xyes;then
-    AM_CONDITIONAL(HAS_CUTTER, true)
-  fi
-fi
-
-fi
-
-#################################################################
 # Check for pcre - perl compatible regular expressions
 #################################################################
 
@@ -358,7 +334,6 @@ libmp3splt.pc
 src/Makefile
 po/Makefile.in
 plugins/Makefile
-test/Makefile
 m4/Makefile
 doc/Makefile
 doc/Doxyfile_all
