Index: libmp3splt/configure.ac
===================================================================
--- libmp3splt.orig/configure.ac
+++ libmp3splt/configure.ac
@@ -131,7 +131,7 @@ AC_MSG_RESULT($HOST)
 AC_ARG_ENABLE(c-debug, [AC_HELP_STRING([--enable-c-debug],[ Enable debugging symbols. ]) ],
     [enable_c_debug=$enableval],[enable_c_debug="no"])
 if test "x$enable_c_debug" = xyes;then
-  CFLAGS="$CFLAGS -g -Wall"
+  CFLAGS="$CFLAGS -g -Werror"
 fi
 
 AC_ARG_ENABLE(optimise, [AC_HELP_STRING([--disable-optimise],[ Disable O3 optimise. ]) ],
